.gitignore

```diff
@@ -0,0 +1,28 @@
+class buttonCommandsCanvasSet(Canvas):
```

Comment:
Why did you add those to the gitignore. gitignore is for another purpose. We will use it now